astropy/io/fits/card.py

```diff
@@ -1299,20 +1299,21 @@ def _format_value(value):
 
 def _format_float(value):
     """Format a floating number to make sure it gets the decimal point."""
-    value_str = f"{value:.16G}"
-    if "." not in value_str and "E" not in value_str:
-        value_str += ".0"
-    elif "E" in value_str:
+    value_str = str(value)
+
+    if "e" in value_str:
         # On some Windows builds of Python (and possibly other platforms?) the
```

Comment:
I doubt this is still true.

Reply:
Do you mean the comments regarding the Windows builds of Python?